website/developer-docs/docs/templates/procedural.md

```diff
@@ -3,16 +3,16 @@ title: "Procedural topic"
 ---
 
 :::info
-**How to use this template**: start with the [markdown version](./procedural.tmpl.md) of the template, either copying the file from the local repo or download the template file using the following command:
+**How to use this template**: start with the markdown version of the template, either by copying the [`procedural.tmpl.md`](https://github.com/goauthentik/authentik/tree/main/website/developer-docs/docs/templates) file from our GitHub repo or downloading the template file using the following command:
```

Comment:
I purposefully didn't link the file here as you'll already have the repo checked out at this point (or you might not but you need to clone it to make a PR), so it doesn't really make sense to wget it

Reply:
Hmmm... I see them as two diff ways to get the raw .md file onto your local machine. I see your point that in order to eventually do a PR, they will need to clone the whole repo (including this file) but... this allows them to first get the tempalte, try out writing, and then decide if they want to go ahead and submit a PR.

My thinking is let's get a template into their hands asap, the easiest way(s). ;-)